packages/fast-development-site-react/src/components/site/menu.tsx

```diff
@@ -86,15 +182,38 @@ class SiteMenu extends React.Component<ISiteMenuProps & IManagedClasses<ISiteNav
         );
     }
 
+    componentDidMount() {
+        this.navPaneElement.current.style.display = "none";
```

Comment:
Can we add a quick comment to detail why we are setting this here? I know, and you know - but will others know implicitly?

Reply:
chatted with @nicholasrice we don't need the component did mount at all